web/settings/modulconfiglp.php

```diff
@@ -1123,6 +1123,32 @@ function visibility_kia_soccalclp1() {
 									</div>
 								</div>
 							</div>
+							<div id="socmintervall1" class="hide">
```

Comment:
Analog zu den gewählten Variablennamen sollten auch die Eingabefelder nicht in separate DIV Elemente gepackt werden, sondern in das Element des Http SoC.